superset-frontend/src/views/CRUD/data/query/QueryList.tsx

```diff
@@ -128,7 +132,7 @@ function QueryList({ addDangerToast, addSuccessToast }: QueryListProps) {
     ...commonMenuData,
   };
 
-  const initialSort = [{ id: 'changed_on', desc: true }];
+  const initialSort = [{ id: 'start_time', desc: true }];
```

Comment:
nit: make this an enum `initSortId` at the top

Reply:
I converted all the column names to an enum